webapp/channels/src/types/global.d.ts

```diff
@@ -2,3 +2,9 @@
 // See LICENSE.txt for license information.
 
 declare const COMMIT_HASH: string;
+declare interface Error {
```

Comment:
What is this change for? I don't see this being used in your changes, but if we do need this, there's a `ServerError` type in `@mattermost/types/errors` that has these fields

Reply:
@hmhealey In the lines where we create a new error object, I was getting an error that `server_error_id` is not there in `Error` Type, that is why I added it.

Reply:
That object can probably be changed to be a `ServerError` then. It's mostly the same as `Error` but it has some MM-specific fields like `server_error_id`. I don't think you should have to change the built-in type for this

Reply:
Ok, I will work on it

Reply:
Should i create a new interface called `ServerError` or is it already present in the mattermost?